test/functional/interface_zmq.py

```diff
@@ -177,7 +179,7 @@ def test_basic(self):
         self.restart_node(0, ["-zmqpubrawtx=foo", "-zmqpubhashtx=bar"])
         self.zmq_context = zmq.Context()
 
-        address = 'tcp://127.0.0.1:28332'
+        address = f"tcp://127.0.0.1:{self.zmq_port_base}"
```

Comment:
26805: we should apply same changes to `interface_zmq_dash.py`
```
class DashZMQTest (DashTestFramework):
    def set_test_params(self):
        # That's where the zmq publisher will listen for subscriber
        self.address = "tcp://127.0.0.1:28331"
```
Otherwise backport 26805 is not done

Reply:
Feel free to do it in another PR I think. I'm gonna merge this one

Reply:
change title to partial?